ligo/src/treasuryDAO.mligo

```diff
@@ -108,8 +101,8 @@ let receive_xtz_entrypoint (params, config_store : bytes * configured_storage)
 // Storage Generator
 // -------------------------------------
 
-let default_treasury_DAO_full_storage (admin, token_address, contract_extra
-    : (address * address * treasury_contract_extra)) : full_storage =
+let default_treasury_DAO_full_storage (admin, token_address, contract_extra, metadata
+    : (address * address * treasury_contract_extra * (string, bytes) big_map)) : full_storage =
```

Comment:
Let's add a `metadata_map` type synonym to `ligo/src/types.mligo` and use that instead.

Also, this change is missing from the `defaults.mligo` as well as the `trivialDAO_storage` in the `Makefile` that uses it.

P.s. remember to update the README as well.

Reply:
Changes done.